runtime/doc/syntax.txt

```diff
@@ -4866,10 +4866,9 @@ Menu		Current font, background and foreground colors of the menus.
 		Also used for the toolbar.
 		Applicable highlight arguments: font, guibg, guifg.
 
-		NOTE: For Motif and Athena the font argument actually
-		specifies a fontset at all times, no matter if 'guifontset' is
-		empty, and as such it is tied to the current |:language| when
-		set.
+                NOTE: For Motif the font argument actually specifies a fontset
+                at all times, no matter if 'guifontset' is empty, and as such
+                it is tied to the current |:language| when set.
```

Comment:
To match what's done in Vim's documentation, tabs should be used for indentation (spaces are used in this block).